src/codec/h264/nalu.rs

```diff
@@ -80,10 +81,10 @@ where
 
         Ok(Nalu {
             header: hdr,
-            data: bitstream,
+            data: Cow::from(&bitstream[start_code_offset..nalu_offset + nal_size]),
             size: nal_size,
-            offset: nalu_offset,
-            sc_offset: start_code_offset,
+            offset: nalu_offset - start_code_offset,
+            sc_offset: 0,
```

Comment:
This looks unrelated to the switch to `Cow`. Actually, this looks more like it should be part of the next commit that removes `sc_offset`?

Reply:
Yeah, It it was related to the comment below. But I've moved to the next commit.